formicaio/umbrel-app.yml

```
@@ -0,0 +1,29 @@
+manifestVersion: 1
+id: formicaio
+name: Formicaio
+tagline: Il potere delle formiche
+icon: ""
+category: social
+version: "0.3.0"
```

Update to latest version.
```suggestion
version: "0.4.9"
```